diff --git a/node_modules/protobufjs/src/util/minimal.js b/node_modules/protobufjs/src/util/minimal.js
index 3c406de..59880a0 100644
--- a/node_modules/protobufjs/src/util/minimal.js
+++ b/node_modules/protobufjs/src/util/minimal.js
@@ -41,7 +41,8 @@ util.isNode = Boolean(typeof global !== "undefined"
  * @memberof util
  * @type {Object}
  */
-util.global = util.isNode && global
+util.global = typeof globalThis !== "undefined" && globalThis
+	   || util.isNode && global
            || typeof window !== "undefined" && window
            || typeof self   !== "undefined" && self
            || this; // eslint-disable-line no-invalid-this
@@ -280,13 +281,13 @@ function newError(name) {
             merge(this, properties);
     }
 
-    (CustomError.prototype = Object.create(Error.prototype)).constructor = CustomError;
+    Object.defineProperty(CustomError.prototype = Object.create(Error.prototype), "constructor", { value: CustomError });
 
     Object.defineProperty(CustomError.prototype, "name", { get: function() { return name; } });
 
-    CustomError.prototype.toString = function toString() {
+    Object.defineProperty(CustomError.prototype, "toString", { value: function toString() {
         return this.name + ": " + this.message;
-    };
+    } });
 
     return CustomError;
 }
